inst/shiny/DiagnosticsExplorer/global.R

```diff
@@ -39,8 +39,8 @@ if (enableAuthorization) {
 
 if (exists("shinySettings")) {
   writeLines("Using settings provided by user")
-  connectionDetails <- shinySettings$connectionDetails
```

Comment:
This change has broken the ability to launch shiny app by using the launchApp button in the R studio UI

Reply:
![image](https://user-images.githubusercontent.com/13936600/163081898-6b179da1-93dd-49cd-b173-d2a9f9f600af.png)

Reply:
Should be fixed in latest push